docs: update KOF UI sections (#554)
k0rdent/docs: docs/admin/kof/kof-using.md

The KOF UI also allows you to monitor internal telemetry from OpenTelemetry collectors and VictoriaMetrics/Logs, enabling comprehensive observability of their health and performance.

![kof-ui-collectors-metrics](../../assets/kof/ui_vm_and_collectors_metrics.gif)

> Resources such as:
> 
> * ClusterDeployment
> * SveltosCluster
> * ClusterSummaries
> * MultiClusterService
> * StateManagementProvider
> * ServiceSet
> 
> Are monitored by the UI. If there are any errors, they will be displayed. These resources help you identify and debug issues in deployed clusters.

Reviewer: * Let's avoid "Are monitored by the UI." grammar,
* sort the list A-Z for quick scanning,
* and simplify wording:
```suggestion
To identify and debug issues in deployed clusters,
check if KOF UI shows any errors in these monitored resources:

* ClusterDeployment
* ClusterSummaries
* MultiClusterService
* ServiceSet
* StateManagementProvider
* SveltosCluster
```
